Classes/EventStore/EventSourcedRepository.php

```diff
@@ -0,0 +1,116 @@
+<?php
+namespace Neos\Cqrs\EventStore;
```

Comment:
Sure, go for it - it may be `Neos\Flow\EventStore` at some point. Or even `Neos\EventSore` ... but it doesn't really matter what it's now during development.